src/components/checkbox/CheckboxGroup.vue

```diff
@@ -11,16 +11,13 @@ export default {
     type: {
```

Comment:
If `:type="type"` is removed, can we remove these lines 11-13?

Reply:
@andrewtpham Yes, thay can be removed, they are not used in that file.

Reply:
Also according to the [element](https://element.eleme.io/#/en-US/component/checkbox) docs, checkbox group component does not have a type as a prop.

Reply:
Thanks both :) I have removed the `type` props